client/src/App.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
+import { Route, BrowserRouter as Router } from 'react-router-dom';
 import { decode } from 'jsonwebtoken';
 
 import Navbar from './components/Navbar';
 import Home from './components/Home';
+import GameBoardComponent from './components/gameBoardComponent/gameBoardComponent';
```

Comment:
Two things, 

1. File name for component should be capitalized. 
2. `components/gameBoardComponent/gameBoardComponent.js` is a bit redundant. you could either use 

`components/GameBoardComponent.js` or `components/GameBoardComponent/index.js`